frontend/models/product-list/component/product-list-redirect-to-type.ts

```diff
@@ -0,0 +1,6 @@
+import { z } from 'zod';
+
+export const ProductListRedirectToType = z.enum(['Permanent', 'Temporary']);
+export type TProductListRedirectToType = z.infer<
```

Comment:
For zod schemas there's a convention of using the `Schema` suffix. This allows you to keep a clean name for the type

```suggestion
export const ProductListRedirectToTypeSchema = z.enum(['Permanent', 'Temporary']);
export type ProductListRedirectToType = z.infer<
```